src/controllers/queries/__tests__/index.test.ts

```diff
@@ -170,7 +171,8 @@ describe('QueriesController', () => {
               action: 'success',
               partialQuery: ['on'],
               productIds: ['product-1', 'product-2'],
-              quantities: ['10', '20'],
+              quantities: [10, 20],
+              emissions: 10,
```

Comment:
if this wasn't in before, what does it do?

Reply:
This is our contribution to the emissions which should be added to whatever comes from the cascaded queries